docs: Add a contribution guide (#13)
asetalias/amibot: Contributing/CONTRIBUTING.md

- Using the command line:
  - [Fork the repo](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo#fork-an-example-repository) so that you can make your changes without affecting the original project until you're ready to merge them.

<!-- 3. Install or update to **Node.js v16**. For more information, see [the development guide](contributing/development.md). -->

3. Create a working branch and start with your changes!

### Project setup

 This describes the setup you need to test and run this application on your local computer.
 
 1. [WhatsApp token](https://developers.facebook.com/docs/whatsapp/business-management-api/get-started) 

- [Setup a facebook developer account](https://developers.facebook.com/)
- [Get started with WhatsApp Business Cloud API](https://developers.facebook.com/docs/whatsapp/business-management-api/get-started)
- Add a test number.

2. [MongoDB](https://www.mongodb.com/)

-[Create a DB on MongoDB Atlas](https://www.mongodb.com/docs/atlas/getting-started/) or create one locally.

> 3. [Node.js](https://nodejs.org/en/download/)
> 
> -Node version (18+ -- use [`nvm`](https://github.com/nvm-sh/nvm) for easy setup)

Reviewer: ```suggestion
3. [Node.js](https://nodejs.org/en/download/)

	- Node version (18+ -- use [`nvm`](https://github.com/nvm-sh/nvm) for easy setup)
```

4. Make `.env` file
-Make `.env` file by copying `.env.sample` and populating whatsapp token, `VERIFY_TOKEN` (for Whatsapp Webhook) and Mongodb details.

5. Start project with `yarn start` or `yarn dev`.

6. Install `ngrok`

> - Create account on https://ngrok.com, login and expose local the local Amibot instance with `ngrok http 3000 --region us` (the region is important -- facebook currently blocks other ngrok regions.

Reviewer: ```suggestion
	- Create account on https://ngrok.com, login and expose local the local Amibot instance with `ngrok http 3000 --region us` (the region is important -- facebook currently blocks other ngrok regions.
```

- Enable the checkbox to [allow maintainer edits](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) so the branch can be updated for a merge.
Once you submit your PR, a Docs team member will review your proposal. We may ask questions or request for additional information.
- We may ask for changes to be made before a PR can be merged, either using [suggested changes](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request) or pull request comments. You can apply suggested changes directly through the UI. You can make any other changes in your fork, then commit them to your branch.
- As you update your PR and apply changes, mark each conversation as [resolved](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#resolving-conversations).
- If you run into any merge issues, checkout this [git tutorial](https://github.com/skills/resolve-merge-conflicts) to help you resolve merge conflicts and other issues.

### Your PR is merged!

> Congratulations :tada::tada: The Amibot team thanks you :sparkles:. 

Reviewer: ```suggestion
Congratulations :tada::tada: The Amity student body thanks you :sparkles:
```
